modules/lightning_features/lightning_core/modules/lightning_search/css/entity-search.css

```diff
@@ -0,0 +1,3 @@
+.ui-autocomplete-loading[data-entity-search] {
```

Comment:
core/modules/system/css/components/autocomplete-loading.module.css around line 16 has the same style defined. Why doesn't our form pick up on that style?